recipes/antismash/meta.yaml

```diff
@@ -16,18 +16,18 @@ source:
 
 requirements:
   host:
-    - python >=3
+    - python >=3.6
     - pip
   run:
-    - python >=3
+    - python >=3.6
```

Comment:
Please make the Python dependencies to be `python >=3.5` since that is what the documentation says.